openbb_terminal/helper_funcs.py

```diff
@@ -1703,3 +1703,40 @@ def get_ordered_list_sources(command_path: str):
         )
         console.print(f"[red]{e}[/red]")
         return None
+
+
+def search_wikipedia(expression: str) -> None:
+    """
+    Search wikipedia for a given expression"
+    Parameters
+    ----------
+    expression: str
+        Expression to search for
+    """
+
+    url = f"https://en.wikipedia.org/api/rest_v1/page/summary/{expression}"
+
+    response = requests.request("GET", url, headers={}, data={})
+    status = response.status_code
```

Comment:
These are minor but:
* No need to declare status variable since you are only checking it once on that if condition
* No need to declare response as response.text, just `json.loads(response.text)`

Reply:
^^^
same here!